### ISSUE TITLE
where is the dataset file 

### ISSUE THREAD

**Issue by ZhouAo-ZA** [pallashadow/SuperGlue-pytorch#1]
@pallashadow 
Thank you for your share!
I want to run your code, but there is no file named dataset

from the dataset.data_builder import DataBuilder

how can I run your training code, and which dataset you have try, do you know how to evaluate it because I want to know the precision and ms like match_pair do

**Comment by pallashadow** [comment]
Sorry I forgot to push the folder.

**Comment by pallashadow** [comment]
It works well on a picture book retrieval and mapping task. But I haven't test it on any SLAM or academic datasets.

**Comment by ZhouAo-ZA** [comment]
> It works well on a picture book retrieval and mapping task. But I haven't test it on any SLAM or academic datasets.

Thank you for your reply. I would like to know how to test and evaluate this model after it is trained, whether to use loss, or to look at the number of matching points？
 Is it convenient for you to share the code for testing and evaluation?

**Comment by ZhouAo-ZA** [comment]
兄弟，自己人啊，那我就不写英文了，方便加我个qq好友吗，像你请教点superglue的问题，多谢了，425109310

**Comment by pallashadow** [comment]
木有QQ

**Comment by ZhouAo-ZA** [comment]
> 木有QQ

。。。那什么方式方便呢，要不给我个邮箱呗，在这里聊不大好

**Comment by pallashadow** [comment]
luyuhao@uzoo.cn
